common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -643,17 +634,71 @@ def get_problem_html(self, encapsulate=True):
         demand_hints = self.lcp.tree.xpath("//problem/demandhint/hint")
         demand_hint_possible = len(demand_hints) > 0
 
+        # Get the current problem status and generate the answer notification and message.
+        answer_notification_message = None
+        answer_notification_type = None
+
+        progress = self.get_progress()
+        id_list = self.lcp.correct_map.keys()
```

Comment:
The error mentioned was that the self.correct_map, was empty, while the self.lcp.correct_map had valid data.